backend/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
+    "mobile-detect": "^1.4.5",

```

Comment:
this has to go into the `backend/cloud/package.json` instead.

I know this is a bit intransparent... `cloud` is what we deploy to back4app and everything else in `backend` is just to wrap the features it provides locally.... but that means we need to tell `back4app` to install `mobile-detect`...